LoopKitUI/View Controllers/DismissableHostingController.swift

```diff
@@ -8,8 +8,11 @@
 
 import SwiftUI
 
-
 public class DismissibleHostingController: UIHostingController<AnyView> {
+    public static var cobTintColor: Color = .green
+    public static var glucoseTintColor: Color = Color(.systemTeal)
+    public static var warningColor: Color = .warning
```

Comment:
[nit, similar to above] I know these colors get used together a lot, so I'm wondering if it could make sense to have a `struct` that could contain all 3.

Reply:
These are ones that I would think may make more sense to keep separate.